src/zkevm_specs/state_circuit.py

```diff
@@ -171,49 +206,54 @@ def assert_in_range(x: FQ, min_val: int, max_val: int) -> None:
 
 @is_circuit_code
 def check_start(row: Row, row_prev: Row):
+    print(row)
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/390/commits/82919ed109795b9c575182f2703ba268dfd0f91e